funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -457,7 +458,11 @@ def migrate_tasks_to_internal(self, kill_event, status_request=None):
                     }
                 )
                 self.total_pending_task_count += 1
-                ts = (time.monotonic(), TaskState.WAITING_FOR_NODES)
+                ts = TaskTransition(
+                    timestamp=time.time_ns(),
+                    state=TaskState.WAITING_FOR_NODES,
+                    actor=ActorName.INTERCHANGE,
+                )
                 self.task_status_deltas[msg.task_id] = ts
```

Comment:
FWIW, `ts` was short for `timestamp` at the time.  That concept slightly changes with this PR, so consider renaming the variable as you see fit ...